src/seq/fasta.rl

```diff
@@ -0,0 +1,251 @@
+
+
+# FASTA sequence types
+
+@doc """
+Metadata for FASTA sequence records containing just a `description` field.
+""" ->
+type FASTAMetadata
+    description::String
```

Comment:
Please don't take my comment seriously. My anxiety would be trivial.
I really approve this pull request.